libredex/IRTypeChecker.h

```diff
@@ -32,7 +32,10 @@ class IRTypeChecker final {
   // definition must be located after the definition of TypeInference.
   ~IRTypeChecker();
 
-  explicit IRTypeChecker(DexMethod* dex_method, bool validate_access = false);
+  explicit IRTypeChecker(DexMethod* dex_method,
+                         bool validate_access = false,
+                         std::unordered_set<std::string> class_skip_list = {
+                             ""});
```

Comment:
Why the empty string?

Reply:
Artifact of trying to get this to work. No real reason to keep.